### DISCUSSION TITLE
Combine rheoInterFoam and rheoHeatFoam

### DISCUSSION THREAD

**Discussion by kenihua** [fppimenta/rheoTool#92] [Q&A]
Hi, 
I want to simulate a two-phase flow with heat transfer. I tried to combine these two solvers,  rheoInterFoam and rheoHeatFoam, however, I met some problems with using fluidThermoMixtureModel.

createFields.H:5:102: error: ‘phase1’ was not declared in this scope
 autoPtr<fluidThermoMixtureModel> thermo(fluidThermoMixtureModel::New(word("thermo"+mesh.name()),mesh,phase1,phase2));
                                                                                                      ^
createFields.H:5:109: error: ‘phase2’ was not declared in this scope
 autoPtr<fluidThermoMixtureModel> thermo(fluidThermoMixtureModel::New(word("thermo"+mesh.name()),mesh,phase1,phase2));

Do you have any idea how to fix it? 

Or any suggestion to generate a solver for two-phase flow with heat transfer

Thanks a lot.

Regards,

Keni

**Comment by nikoscham** [comment]
I am wondering if you have found a solution to your problem. I am also interested in adding heat transfer to rheoInterFoam.
